DM-29731: implement RFC-695 and LCR-2722

MelissaGraham opened the pull request:
Update baselineSchema.yaml

Updates to the DIAObjects table to reflect the updated plans for host association as described in LCR-2722
https://project.lsst.org/groups/ccb/node/4612

Added new host association parameters nearbyExtObj1 (& 2,3), nearbyExtObj1Sep (& 2,3), nearbyLowzGal, and nearbyLowzGalSep. New rows 609-652. "Sep" (separation) values further described in footnotes of Table 3 of LSE-163.

womullan reviewed (commented):
shouldn't this have units as well arcsec ?   line fits/ltunit/arcsec

womullan reviewed (approved):
this matches the document units not withstanding.  (does need a rebase though)

MelissaGraham commented:
Based on Wil's comments:
Added "(unitless)" to the description of nearbyExtObj1Sep (and to 2 and 3).
Fixed typo in that nearbyExtObj1 was repeated when it should have been 3.
Added "fits:tunit arcsec" to nearbyLowzGalSep.
